views/build.html

```diff
@@ -7,64 +7,65 @@
     <dd>{{resource.metadata.namespace}}</dd>
     <dt>Created</dt>
     <dd>{{resource.metadata.creationTimestamp | date:'medium'}}</dd>
-  </dl>  
+  </dl>
   <h3>Build Configuration</h3>
   <dl class="dl-horizontal" style="margin-bottom: 0;">
     <dt>Strategy</dt>
-    <dd>{{resource.parameters.strategy.type}}</dd>
-  </dl>    
-  <div ng-switch="resource.parameters.strategy.type">
-    <div ng-switch-when="STI">
-      <div ng-if="resource.parameters.strategy.stiStrategy.image">
+    <dd>{{resource.spec.strategy.type}}</dd>
+  </dl>
+  <div ng-switch="resource.spec.strategy.type">
+    <div ng-switch-when="Source">
+      <!-- FIXME: Is this right? -->
+      <div ng-if="resource.spec.strategy.sourceStrategy.image">
         <dl class="dl-horizontal" style="margin-bottom: 0;">
           <dt>Builder image</dt>
-          <dd>{{resource.parameters.strategy.stiStrategy.image}}</dd>
+          <dd>{{resource.spec.strategy.sourceStrategy.image}}</dd>
```

Comment:
I think it's `...sourceStrategy.from`, which is the structured object reference discussed above